pkgs/applications/misc/bottles/default.nix

```diff
@@ -55,8 +55,9 @@ python3Packages.buildPythonApplication rec {
     gobject-introspection
     gsettings-desktop-schemas
     gspell
-    gtk3
-    gtksourceview4
+    gtk4
+    gtksourceview5
+    libadwaita
     libhandy
```

Comment:
Libhandy is not required: https://github.com/bottlesdevs/Bottles/wiki/Packaging

Reply:
Oh thanks, will check against it!